sections/jobs.zsh

```diff
@@ -20,11 +20,15 @@ SPACESHIP_JOBS_COLOR="${SPACESHIP_JOBS_COLOR="blue"}"
 spaceship_jobs() {
   [[ $SPACESHIP_JOBS_SHOW == false ]] && return
 
-  local jobs_amount=$( (jobs) | wc -l )
+  local jobs_amount=$( (jobs) | wc -l | xargs)
 
   [[ $jobs_amount -gt 0 ]] || return
   [[ $jobs_amount -eq 1 ]] && jobs_amount=''
 
+  if [[ $jobs_amount -gt 1 ]]; then
+    jobs_amount=" ${jobs_amount}"
```

Comment:
With my symbol it looks better without a space, maybe instead of doing this append a space to `SPACESHIP_JOBS_SYMBOL` in your config?

Reply:
This is the output of jobs without appending a space. Seems odd to me. 

![image](https://user-images.githubusercontent.com/2712115/40699066-a6532090-63a8-11e8-9349-ddcd5760155e.png)

What do you think? Should I keep the space or remove it?

Reply:
Let's make it optional, have or not space before the job count. Maybe call it `SPACESHIP_JOBS_AMOUNT_PREFIX` and set it to empty string by default, not to change behavior for existing users.

Here's how it looks for me:

![image](https://user-images.githubusercontent.com/1177900/40706910-70e071dc-63ef-11e8-9022-3cb5fb491d80.png)

Reply:
Done! This last commit also fixes #272